Calling/src/app/views/CallScreen.tsx

```diff
@@ -76,6 +81,7 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
   // This ensures the service knows the user intentionally left the call if the user
   // closed the browser tab during an active call.
   useEffect(() => {
+    adapter?.joinCall();
```

Comment:
you need to make sure we still call joinCall. This is normally called when we hit the start call button on the config screen but since we are skipping the config screen then we need to call joinCall when the component mounts